Assets in .css files are breaking formidable-charts-docs

Using a relative path (e.g. `background: url("../../static/hero.png;")` causes the `build-static` task to break in `formidable-charts-docs`, whereas an absolute path does not (e.g. `background: url("/static/hero.png;")`. Unfortunately this particular file in question is not small enough to be a data uri, which means the absolute path breaks in production: `/static/hero.png` does not exist, as the path should be `/open-source/victory/static/hero.png`.

Not sure how to resolve it yet—maybe a different webpack loader? maybe need a different postcss plugin to import `.png` (like how we’re using `postcss-inline-svg` for `.svg` files)? I recently read `css-loader` should be using `require()` to resolve the path name, but webpack can’t find it when it’s being pulled in as a dependency is my current theory

Error: https://travis-ci.org/FormidableLabs/formidable-charts-docs/jobs/175266161#L1682

```
modulesModuleNotFoundError: Module not found: Error: Cannot resolve 'file' or 
'directory' ../../static/hero@2x.png in 
/home/travis/build/FormidableLabs/formidable-charts-docs/src/styles
```

--- a/config/webpack/webpack.config.base.js
+++ b/config/webpack/webpack.config.base.js
@@ -2,6 +2,7 @@
 
 var path = require("path");
 var postcssImport = require("postcss-import");
+var postcssUrl = require("postcss-url");
 var postcssNext = require("postcss-cssnext");
 var postcssInlineSvg = require("postcss-inline-svg");
 
@@ -53,6 +54,7 @@ module.exports = {
   postcss: function (webpack) { // eslint-disable-line no-shadow
     return [
       postcssImport({ addDependencyTo: webpack }),
+      postcssUrl,
       postcssNext,
       postcssInlineSvg
     ];

--- a/package.json
+++ b/package.json
@@ -41,6 +41,7 @@
     "postcss-import": "^8.1.2",
     "postcss-inline-svg": "^2.2.0",
     "postcss-loader": "^0.11.1",
+    "postcss-url": "^5.1.2",
     "raw-loader": "0.5.1",
     "static-site-generator-webpack-plugin": "2.1.0",
     "style-loader": "^0.13.1",

